pyproject.toml

```
@@ -22,6 +22,6 @@ dependencies = [
     "mediaflow-proxy",
     "orjson",
     "pydantic-settings",
-    "rank-torrent-name==1.7.2",
+    "rank-torrent-name==1.8.2",
```

```suggestion
    "rank-torrent-name~=1.8.2",
```

was only wondering if that makes sense maybe to get future updates? not sure what the best practices in python are, but I'm normally used to using ranges like that to a) unintentional avoid major updates and b) not be locked to a single version forever :)

---

RTN should be pinned for sure because new updates could break settings. So the developers that use it should be manually making those updates.

RTN was created for Riven, so I only change it when I need to add features for use in Riven, but that may break functionality in other apps that use the library.

---

makes sense, thx!